examples/md-flexible/src/Simulation.h

```diff
@@ -199,6 +215,12 @@ class Simulation {
    */
   std::string timerToString(const std::string &name, long timeNS, size_t numberWidth, long maxTime);
```

Comment:
```suggestion
  std::string timerToString(const std::string &name, long timeNS, size_t numberWidth = 0ul, long maxTime = 0ul);
```
Used to have default args for convenience.